src/api/migration/database.ts

```diff
@@ -0,0 +1,41 @@
+import { createConnection, Connection } from 'typeorm';
```

Comment:
@bigLucas 
It was this format you expected? 
I used PF project as example.

Reply:
@Aristevao It's a little different than I thought...
We can improve this, I will test some points and after it I will tell you how we can improve!